### ISSUE TITLE
Support for Web Content Accessibility Guidelines (WCAG) 2.1

### ISSUE THREAD

**Issue by faust347** [AndrewEC/ContrastCalculator#1]
The latest binary release is from 2017, so it cannot include the WCAG 2.1 guidelines which are from 2018. The calculation of the acceptable contrast ratio has slightly changed, it would be good to have it reflected in your application. Otherwise: congratulations for a really useful tool.
